### PR TITLE
feat: Adding missing integrations and info for people integrating their services with Apify

### PR DESCRIPTION
**Pull request by mtrunkat**
- Added missing integrations to https://docs.apify.com/platform/integrations
- Removed + redirected GPT plugin
- Added info for 3rd SaaS apps that want to integrate with us

@jancurn adding you as GPT plugin was yours and also as it was your idea to add this page. Once done content-wise, Ill add Theo.

### CODE REVIEW

File: sources/platform/integrations/index.mdx [hunk 1]
```diff
@@ -120,6 +122,12 @@ If you use one of the main automation platforms, Apify support is here for you.
         imageUrl="/img/platform/integrations/transposit.svg"
         smallImage
     />
```
Comment (mtrunkat) [inline sources/platform/integrations/index.mdx:124]: This was missing

File: sources/platform/integrations/index.mdx [hunk 2]
```diff
@@ -158,6 +166,18 @@ If you are working on an AI/LLM-related project, we recommend you look into the
         imageUrl="/img/platform/integrations/langchain.png"
         smallImage
     />
```
Comment (mtrunkat) [inline sources/platform/integrations/index.mdx:168]: This was missing

File: sources/platform/integrations/integrate_with_apify.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (mtrunkat) [inline sources/platform/integrations/integrate_with_apify.md:1]: Info for 3rd SaaS services that would like to integrate with us

File: nginx.conf [hunk 4]
```diff
@@ -286,4 +286,8 @@ server {
   rewrite ^/platform/actors/development/continuous-integration$  /platform/actors/development/deployment/continuous-integration redirect;
   rewrite ^/platform/actors/development/testing-and-maintenance$ /platform/actors/development/deployment/automated-tests redirect;
   rewrite ^/platform/actors/development/source-code$             /platform/actors/development/deployment/source-types redirect;
+
+  # Removed pages
```
Comment (mtrunkat) [inline nginx.conf:290]: Removing GPT plugins as they are about to stop working today - @jancurn do we have a better redirect?

Comment (jancurn) [reply nginx.conf:290]: IMO this is good, cheers

**Review by jancurn — APPROVED** [review]
Great stuff Mara

**Comment by mtrunkat** [comment]
@TC-MO I am fighting with links. Could you help me with that? I can't figure out what is wrong as it works locally. Thanks!

**Comment by TC-MO** [comment]
@mtrunkat the issue with links seems to be fixed. Seems it was some Docusaurus hiccup that prevented from even builiding the site.

Additionally I fixed some typos that I've spotted and changed structure a bit.

Maybe this should be first page in the left-nav under `Integrations` instead of last?

Also Keboola integration doc is in review, maybe we can wait with this one until it is merged and change the link to our docs?

**Comment by TC-MO** [comment]
I see now that Lychee is failing, but there are technically no broken links. 

Since we are checking everything against `baseURL` it tries to check the newly created page as well thus throwing errors, I encountered similar issue on the new Keboola integration doc

**Review by TC-MO — APPROVED** [review]

**Pull request merged**